examples/image-to-text/run_pipeline.py

```diff
@@ -174,9 +174,15 @@ def main():
         action="store_true",
         help="Whether to use the key/value cache for decoding. It should speed up generation.",
     )
+    parser.add_argument(
```

Comment:
--sdp_on_bf16 is not added to README. should I update all of the model in image-to-text also?

Reply:
im only setting for "meta-llama/Llama-3.2-11B-Vision-Instruct"  and  "tiiuae/falcon-11B-vlm", so lets do only for those.

Reply:
this file change is already added by https://github.com/huggingface/optimum-habana/commit/89dfbb7b3ba1dd2a6a96e3f9a796d0d92bfdeea0